Swift3

hryk224 opened the pull request.

hryk224 committed:
Support swift3

hryk224 committed:
Support ios8

imkashrao commented:
Hi There, I want to update from 2.3 to 3.0 swift. What would be the pod for the Swift 3? Thanks.

rafcabezas committed:
Changed delegate call from collectionView to infiniteCollectionView

This removes potential error:
"Objective-C method 'collectionView:didSelectItemAtIndexPath:' provided by method 'collectionView(_:didSelectItemAt:)' does not match the requirement's selector ('collectionView:didSelectItemAt:')"

hryk224 committed:
Merge pull request #7 from AxialExchange/swift3

Changed delegate call from collectionView to infiniteCollectionView

hryk224 committed:
Updated: README, Example

hryk224 committed:
Support orientation changed

hryk224 committed:
Merge pull request #8 from hryk224/feature/swift3_orientaion

Support orientation changed
